src/pages/instances/InstanceOverviewDevices.tsx

```diff
@@ -6,7 +6,7 @@ import { isRootDisk } from "util/instanceValidation";
 import { FormDevice } from "util/formDevices";
 import ResourceLink from "components/ResourceLink";
 import { isOtherDevice } from "util/devices";
-import InstanceOverviewDeviceDetail from "./InstanceOverviewDeviceDetail";
+import InstanceOverviewDeviceDetail from "../../components/OverviewPageDeviceDetails";
```

Comment:
```suggestion
import InstanceOverviewDeviceDetail from "components/OverviewPageDeviceDetails";
```
Better not use relative imports, when crossing module boundaries.

Reply:
Thanks for letting me know. That's what I get for accepting imports to be automatically changed. I'll change them manually from now on.